.github/actions/setup-tebako/action.yml

```diff
@@ -0,0 +1,109 @@
+name: 'Setup tebako'
+description: 'Setup Ubuntu ci environment for tebako'
+inputs:
+  cache:
+    description: 'Tebako cache mode: off/build/use'
+    required: true
+    type: string
+  cache_path:
+    description: 'Tebako cache root folder'
+    required: true
+    default: '.tebako/deps'
+    type: string
+  cache_ver:
+    description: 'Tebako cache version'
+    required: true
+    default: '0'
+    type: string
+  ruby_version:
+    description: 'Ruby version to install, "env" to use RUBY_VER environment variable'
+    required: true
+    default: 'env'
+    type: string
+  bundler_version:
+    description: 'Bundler version to install'
+    required: true
+    default: 'default'
+    type: string
+runs:
+  using: "composite"
+  steps:
+      # FIXME replace it with *.env file read after moving back to tebako
+    - name: Prepare env
+      shell: bash
+      run: |
+        OS=$(uname -s)
+        if [ "$OS" = "Linux" ] && [ -e "/etc/lsb-release" ]; then
+            echo "CC=clang-12" >> $GITHUB_ENV
+            echo "CXX=clang++-12" >> $GITHUB_ENV
+            echo "BUNDLE_BUILD__SASSC=--enable-static-stdlib" >> $GITHUB_ENV
+        elif [ "$OS" = "Darwin" ]; then
+            echo "CC=clang" >> $GITHUB_ENV
+            echo "CXX=clang++" >> $GITHUB_ENV
+            echo "XCODE_VERSION=12.4" >> $GITHUB_ENV
+        elif [ "$OS" = "Linux" ] && [ -e "/etc/alpine-release" ]; then
+            echo "CC=clang" >> $GITHUB_ENV
+            echo "CXX=clang++" >> $GITHUB_ENV
+            echo "VCPKG_FORCE_SYSTEM_BINARIES=1" >> $GITHUB_ENV
+        else
+            echo "Unsupported operating system."
+        fi
+
+        echo "BUILD_TYPE=Release" >> $GITHUB_ENV
+        echo "TEBAKO_DIR=.archive/tebako" >> $GITHUB_ENV
+        echo "INCBIN_TAG=348e36b" >> $GITHUB_ENV
+        echo "DWARFS_WR_TAG=v0.4.0" >> $GITHUB_ENV
```

Comment:
Please do not set ```INCBIN_TAG``` and ```DWARFS_WR_TAG```, it is a rudimant in packed_mn workflow.
```tebako``` will use some reasonable defaults

Reply:
Fixed